### DISCUSSION TITLE
Welcome to astro-fathom Discussions!

### DISCUSSION THREAD

**Discussion by tony-sull** [tony-sull/astro-fathom#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MDBrittingham** [comment]
Hi Tony. I followed your instructions for installing astro-fathom exactly but received an error on compile: ""default" is not exported by "node_modules/astro-fathom/index.ts", imported by "src/layouts/BaseLayout.astro"."  Any insights?

**Reply by MDBrittingham** [reply]
Which is weird because index.tx is literally: "export { default as Fathom } from "./Fathom.astro";"

I am not using typescript in my site. Could that be the problem?

**Reply by MDBrittingham** [reply]
Strange. I just used fathom.astro directly as a component. It compiles just fine but is not included in the build...which is right where I was when I put the call to fathom in my header.

**Reply by MDBrittingham** [reply]
And, perhaps the strangest thing of all - it is working. I cannot find *any* mention of Fathom anywhere nor a link to the site in any of the files being deployed, but it is registering stuff on Fathom. There is obviously a hole in my understanding of Astro, somewhere. Given that I just adopted it two days ago, though, perhaps that's not surprising!  Anyway, thank you Tony!
